lib/svgo/coa.js

```diff
@@ -297,7 +301,7 @@ async function action(args, opts, command) {
   }
 
   // --folder
```

Comment:
``// directory``

Reply:
Zostawiam to dlatego, bo to pochodzi oryginalnie z svgo i oni tak nazwali tą flagę

Reply:
Zostawiłem to, bo jest to flaga pochodząca z oryginały svgo